carfield.mk

```diff
@@ -118,7 +118,7 @@ endif
 ######################
 
 CAR_NONFREE_REMOTE ?= git@iis-git.ee.ethz.ch:carfield/carfield-nonfree.git
-CAR_NONFREE_COMMIT ?= 6d22bcbd9f8c2508bf3089a90003d6e4b3d11e34
+CAR_NONFREE_COMMIT ?= 08feeacb4553465b3587ddfb175f9fed1fdf1faa
```

Comment:
I merged your PR on `carfield-nonfree`; when rebasing this branch on top of carfield main, please update the nonfree pointer as well